apps/examples/SoundPlayer/soundplayer_main.cpp

```diff
@@ -284,15 +290,21 @@ player_result_t SoundPlayer::startPlayback(void)
 	source->setPcmFormat(DEFAULT_FORMAT_TYPE);
 	res = mp.setDataSource(std::move(source));
 	if (res != PLAYER_OK) {
+		handleError((player_error_t)res);
 		printf("set Data source failed. res : %d\n", res);
 		return res;
 	}
 
 	res = mp.prepare();
 	if (res != PLAYER_OK) {
+		handleError((player_error_t)res);
 		printf("prepare failed res : %d\n", res);
 		return res;
 	}
+	if (mLooping) {
```

Comment:
is looping to be done before prepare

Reply:
Yes. This change is already in a pr for ecode